ise-ethereum/on-chain-chess
# Feature/game state factory

## Description

Add game state factory to save local state
## Related Issue

Closes #166 Add states to local storage
## How Has This Been Tested?
- Create and play games against each other
- Reload page to ensure state is in local storage
## Types of changes


## Checklist contributor:




## Checklist reviewer:
- [ ] The code has been peer-reviewed by @YourNameHere
